src/api/coupon.ts

```diff
@@ -0,0 +1,25 @@
+import { getCookie } from '@/libs/manageCookie';
+
+const BASE_URL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
+
+export const getCoupons = async () => {
```

Comment:
별건아니지만
```suggestion
export const getCoupons = async (): Promise<CouponTypes[]> =>
```
저희는 어떤거 오는지 아니까 타입 설정해줘도 좋을거 같아용 ~ 예시입니둥~